docs/Manifest.toml

```diff
@@ -1,5 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
```

Comment:
I thought that Manifests should not be added to Git. (https://github.com/github/gitignore/blob/master/Julia.gitignore)

Reply:
It's the manifest file of the docs only which is quite helpful.

Reply:
Not needed though I reckon

Reply:
I frankly have no opinion on this.